bin/cliain/src/vesting.rs

```diff
@@ -0,0 +1,61 @@
+use aleph_client::{
+    account_from_keypair, keypair_from_string, BlockNumber, Connection, KeyPair, VestingSchedule,
+};
+use log::{error, info};
+use primitives::{Balance, TOKEN};
+
+/// Retrieves signer from `connection`.
+///
+/// Panics if `connection` is not signed.
+fn get_caller(connection: &Connection) -> KeyPair {
```

Comment:
See other comment in `aleph-client` - imho we don't need a separate signer as `Connection` is signed already by `--seed`.

Reply:
+1
Having to do that kind of shaneningans simply shows that `set_signer` does not belong inside `aleph-client` functions

Reply:
And that there should be a separate type - `SignedConnection` that differentiates at the type level.

Reply:
let me handle this here:
https://cardinal-cryptography.atlassian.net/browse/A0-569